SStream.c

```diff
@@ -220,13 +223,15 @@ void SStream_concat(SStream *ss, const char *fmt, ...)
 // print number with prefix #
 void printInt64Bang(SStream *O, int64_t val)
 {
+	assert(O);
```

Comment:
An unfortunate name. Initially I thought it's zero.

Reply:
Yeah, let me rename it.